src/ontology/phenio.Makefile

```diff
@@ -3,4 +3,71 @@
 ## If you need to customize your Makefile, make
 ## changes here rather than in the main Makefile
 
+# Replace the assembly of phenio-full.owl with a
+# similar process including subq reconstruction
 
+# Constants
+MINIMAL_PATH=					$(TMPDIR)/$(ONT)-min.owl
+OT_MEMO=						50G
+OWLTOOLS=						OWLTOOLS_MEMORY=$(OT_MEMO) owltools --no-logging
+RG= 							relation-graph
+SUBQ_QUERY_PATH=                $(SPARQLDIR)/subq_construct.sparql
+SUBQ_QUERY_RESULT_PATH=         $(TMPDIR)/$(ONT)-full_subqs_queryresult.tmp.owl
+UPDATE_QUERY_PATH=              $(TMPDIR)/subq_update.sparql
+
+# Base file assembly
+$(TMPDIR)/$(ONT)-full-unreasoned.owl: $(SRC) $(OTHER_SRC)
+	$(ROBOT) merge --input $< $(patsubst %, -i %, $(OTHER_SRC)) $(patsubst %, -i %, $(IMPORT_FILES)) \
+		--output $@
+
+$(TMPDIR)/$(ONT)-full.owl: $(TMPDIR)/$(ONT)-full-unreasoned.owl
+	# $(ROBOT) reason --input $< \
+	# 	--reasoner ELK --equivalent-classes-allowed all --exclude-tautologies structural \
+	# 	relax \
+	# 	reduce -r ELK \
+	# 	$(SHARED_ROBOT_COMMANDS) annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) --output $@.tmp.owl && mv $@.tmp.owl $@
+	$(ROBOT) relax --input $< \
```

Comment:
If you really just want to do this you have to switch all ontologies to their normal purls and use `make_base` instead. Remember we do not use base-plus in OBO. The `base` files are unreasoned and therefore extremely lossy. It is the expectation that you run the reasoner. So if you want to go this way, you have to change everything to what you do for CHEBI:

```
- id: chebi
   make_base: TRUE
```

Reply:
This will also take care of the uPheno problem for now.

Reply:
OK - if ODK brings in a base version, will it still treat it as an import? That is, will it still attempt to merge in the full set of native axioms for each source ontology? e.g. we want all of ZFA whether it's referenced elsewhere in PHENIO or not.

Reply:
I mean, if you use 

```
- id: upheno
   mirror_from: http://purl.obolibrary.org/obo/upheno/v2/upheno.owl
   make_base: true
```
Then you will get _only_ the uPheno classes. You can use the [base_iris](https://github.com/pato-ontology/pato/blob/master/src/ontology/pato-odk.yaml#L35) parameter to specify which namespace to pull in from uPheno. Note that if you do that, you will add HP, MP, ZP etc (you can find all the IRI spaces [here](https://github.com/obophenotype/upheno-dev/blob/master/src/curation/upheno-config.yaml#L289), just search for `root:` and you should catch them all) as base-uris to this `base_iris` parameter (probably the right approach).